gradle/common/kotlin-jvm.gradle

```diff
@@ -13,6 +13,6 @@ tasks.withType(Jar) { task ->
             'Implementation-Title': 'Spek', \
             'Implementation-Version': rootProject.version, \
             'Implementation-Kotlin-Version': kotlinVersion, \
-            'Build-Time': new Date().format('yyyy-MM-dd\'T\'HH:mm:ssZ')
+            'Build-Time': System.getenv("TEAMCITY_VERSION") ? new Date().format('yyyy-MM-dd\'T\'HH:mm:ssZ') : new Date(0).format('yyyy-MM-dd\'T\'HH:mm:ssZ')
```

Comment:
I will most likely edit it to not directly depend on TeamCity and maybe produce build time with at least "current day" resolution for local builds, 

but I don't wanna block you :)

Reply:
You won't block anything :) 

Just want this to work for everyone first and foremost. Any boolean that indicates that it's a local build (vs CI) will work here. 

BTW, I just ran into this while trying out Kotlin support in Gradle and figured to make a quick fix.